PYCS_Language/Program.cs

```diff
@@ -1,10 +1,13 @@
-﻿namespace SplitAndMerge
+﻿
+namespace SplitAndMerge
 {
     public class Program
     {
         static void Main(string[] args) {
             FileHandler file = new FileHandler();
-            file.ReadFile();
+            string[] text = file.ReadFile();
+            TextParser parser = new TextParser();
+            parser.ParseText(text);
```

Comment:
Please move this logic to a separate class
```
FileHandler file = new FileHandler();
            file.ReadFile();
            string[] text = file.ReadFile();
            TextParser parser = new TextParser();
            parser.ParseText(text);
```

Reply:
Done!